submissions/zadorogniyandrey/scss/style.scss

```diff
@@ -0,0 +1,178 @@
+@import "nullstyle.scss";
+
```

Comment:
Try to use relative units `rem` and `em`, it's a good practice nowadays. 
You can read articles [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units), [here](https://www.freecodecamp.org/news/what-is-rem-in-css/) or any related to it's info

Reply:
I still can see only pixels :)